### PR TITLE
add cheerio and request data from rightToRemain website

### PR DESCRIPTION
**Pull request by khaledkzy**

### CODE REVIEW

**Comment by kabaros** [comment]
Add documentation

File: README.md [hunk 1]
```diff
@@ -131,3 +131,6 @@ We use `knex` for migrations, but we created alias helpers on `package.json` to
 ## To Open Cypress
 
 1. Go to server and run `./node_modules/.bin/cypress open` or `npx cypress open` in terminal and it will open a window.
+
+## To Run the scraper tool
+1. To start the scraper got to server and `npm run scrape-data`
```
Comment (kabaros) [inline README.md]: add more description to what the scrap tool is

File: server/__tools/crawlRightToRemain.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const request = require('request-promise')
+const cheerio = require('cheerio')
+const path = require('path')
+var dir = path.join(__dirname, '../seeds/data')
```
Comment (kabaros) [inline server/__tools/crawlRightToRemain.js]: const

File: server/__tools/crawlRightToRemain.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+var fs = require('fs')
```
Comment (kabaros) [inline server/__tools/crawlRightToRemain.js]: const

File: server/seeds/index.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 const languagesData = require('./data/languages')
 const categoriesData = require('./data/categories')
 const generateArticles = require('./data/articles')
+const rightToRemain = require('./data/rightToremain.json')
```
Comment (kabaros) [inline server/seeds/index.js]: name the file `rightToRemain` not `rightToremain` camelcased

**Comment by khaledkzy** [comment]
[Tool to crawl and get data from Right To Remain website](https://trello.com/c/juiOCezn/329-tool-to-crawl-and-get-data-from-right-to-remain-website)

**Pull request merged**